### DISCUSSION TITLE
Don't announce the persons IP while doing a ban using that persons IP

### DISCUSSION THREAD

**Discussion by iwickdev** [Pryaxis/TShock#2030] [Ideas]
I was banning someone on my server and I went to ban the IP and it announced the persons IP to everyone in chat in the ban message. Maybe blur the IP like 19*.***.*.* or convert to a name?

**Comment by moisterrific** [comment]
I can see this as a config option, why does it matter anyway it’s not like someone can get doxxed from their public IP, right?

**Reply by iwickdev** [reply]
Id just rather not leek peoples IP's every time I need to ban one. There is not much you can to with one other then getting a approximation of there location, some service provider info, etc. But I still would rather not be publicly posting them, also its not my IP.

**Comment by hakusaro** [comment]
I believe you could use a silent command for this right? Does `.ban` not work?
